authors/apps/authentication/tests/test_passwordreset.py

```diff
@@ -0,0 +1,74 @@
+import json
+import jwt
+from datetime import datetime, timedelta
+from django.conf import settings
+from django.urls import reverse
+from django.core import mail
+from rest_framework.views import status
+from rest_framework.test import APITestCase, APIClient
+
+
+class ResetPasswordTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.password_reset_url = reverse('authentication:passwordreset')
+        self.password_update_url = "authentication:passwordresetdone"
+        self.signup_url = reverse('authentication:auth-register')
+        self.signup_data = {
+            "user": {
+                "username": "MaryGigz",
+                "email": "chegemaggie1@gmail.com",
+                "password": "g@_Gigz-2416"
+                }}
+        self.email = {
+            "email": "chegemaggie1@gmail.com"
+        }
+        self.email_not_registered = {
+            "email": "kenna@gmail.com"
+        }
+        self.test_update_password_data = {
+            "password": "Jake@1234",
+            "confirm_password": "Jake@1234"
+
+        }
+
+    def create_url(self, email):
+        """create a url with the token, this is done after user receives an email"""
+        token = jwt.encode({"email": email, "iat": datetime.now(),
+                            "exp": datetime.utcnow() + timedelta(minutes=5)},
+                                    settings.SECRET_KEY, algorithm='HS256').decode()
+        password_reset_url = reverse("authentication:passwordresetdone",
```

Comment:
`"authentication:passwordresetdone"` - this has been declared as a string on line 15, why not use that and keep your code DRY